src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
```

Comment:
Here again, you are using hard-coded values in the calculations. It would be better to use constants instead, to make the code easier to understand and modify. For example, you could define a constant `SHORT_TERM_RENTAL_DAYS = 3`, `SHORT_TERM_DISCOUNT = 20` etc.